lom/src/lib/lomcommon/yang_parser/yang_prod_configs/actions/device-health-actions-common.yang

```diff
@@ -0,0 +1,59 @@
+module device-health-actions-common {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/actions/device-health-actions-common";
+
+    prefix dhac;
+
+    organization
+        "SONiC";
+
+    contact
+        "Device";
+
+    description
+        "Action common configurations for Device health agent plugins";
+
+    revision 2023-06-28 {
+        description
+            "Initial revision.";
+    }
+	
+    grouping detection-action-config-common {
+        leaf Type {
+            type string;
+            description "Indicates Detection plugin";
```

Comment:
Could it be a union of few constant strings ?
This way we avoid any mis-spelling